hamp_radar/flight_dataset.py

```diff
@@ -36,7 +37,14 @@ def read_datasetblock(
 def read_dataset(dsgeom: DatasetGeometry, postprocess: bool):
     ds_ppar = read_datasetblock(dsgeom.ppar)
 
-    dataset = [read_datasetblock(d, ds_ppar).chunk("auto") for d in dsgeom.data]
+    chunks = {
+        "cocx": 2,
+        "range": 512,  # TODO(ALL): see HACK about range dimension = 512
+        "fft": 256,
+        "iq": 2,
+        "frame": 128,  # TODO(ALL): HACK makes iq dask array chunks circa. 100MB
+    }
+    dataset = [read_datasetblock(d, ds_ppar).chunk(chunks) for d in dsgeom.data]
```

Comment:
This is where you define the initial dask chunking. The chunking at this point should be at least close to the desired (`safe_chunks`?) chunksize.

Reply:
Yes exactly what I thought from your last comment too! Will redo.

Reply:
When combining the datasets from each pds file into one final dataset the frame dimension needs to be rechunked (again). I therefore restructed these functions so they better subdivide the tasks.

It looks more complicated than it is because I left in the ability to use dask's "auto" chunks instead of our `ideal_daskchunks` (via the `use_autochunks` boolean) but if you agree I would remove that option.